src/NBitcoin/Block.cs

```diff
@@ -66,24 +67,69 @@ public BlockHeader()
             this.SetNull();
         }
 
-        public BlockHeader(string hex)
-            : this(Encoders.Hex.DecodeData(hex))
+        public BlockHeader(byte[] bytes)
```

Comment:
Not being used.

Reply:
Good point fixed NBitcoin.RPC.RPCClient.ParseBlockHeader to use the consensus factory thanks.